<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataFileView</class>
 <widget class="QGroupBox" name="DataFileView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Data File</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>481</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="descrtab">
    <attribute name="title">
     <string>Description</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Name (optional)</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="descr">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>471</width>
       <height>301</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="vartab">
    <attribute name="title">
     <string>Variables</string>
    </attribute>
    <widget class="QPushButton" name="addbut">
     <property name="geometry">
      <rect>
       <x>239</x>
       <y>292</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add variable</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rembut">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>300</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RemoveAt</string>
     </property>
    </widget>
    <widget class="QLabel" name="varlist">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>471</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="numtodel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>300</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="newvarname">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="newvarval">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>5</number>
     </property>
     <property name="minimum">
      <double>-1000000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="evcnt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>EventCounter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sp2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>SP2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sp1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>SP1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-read source</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
